sqlmesh/utils/__init__.py

```diff
@@ -162,6 +162,19 @@ def nullsafe_join(join_char: str, *args: t.Optional[str]) -> str:
     return join_char.join(filter(None, args))
 
 
+def str_to_bool(s: t.Optional[str] = None) -> bool:
```

Comment:
[Nit] Why have a default value at all?

Reply:
Yeah good point. Will remove the default.